DEVELOPER_GUIDE.md

```diff
@@ -164,6 +164,34 @@ Run tests :
 ```
 ./gradlew clean build test
 ```
+## Generate Artifact
+
+In opensearch-sdk navigate to build/distributions. Look for tar ball in the form `opensearch-sdk-1.0.0-SNAPSHOT.tar`. If not found follow the below steps to create one:
+
+```
+./gradlew clean && ./gradlew build
+```
+Once the tar ball is generated navigate to /src/test/resources and look for extension.yml. Create one if not present
+
+Look for tar ball in /build/distributions. To run the artifact i.e. tar ball, run the below command
+
+```
+tar -xvf opensearch-sdk-1.0.0-SNAPSHOT.tar
+```
+For now, navigate to  opensearch-sdk/build/distributions/opensearch-sdk-1.0.0-SNAPSHOT/ 
+- Check if src folder exists in using `ls`.
```

Comment:
Are we not sure if `src` folder will exist?

Reply:
Yeah we have to create src folder

Reply:
Once https://github.com/opensearch-project/opensearch-sdk/issues/52 this get resolved, there will no need to create src folder as location of extension.yml file will be finalized.